src/main/java/taxi/controller/car/GetMyCurrentCarsController.java

```diff
@@ -0,0 +1,25 @@
+package taxi.controller.car;
+
+import java.io.IOException;
+import java.util.List;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import taxi.dao.CarDao;
+import taxi.lib.Injector;
+import taxi.model.Car;
+
+public class GetMyCurrentCarsController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("taxi");
+    private final CarDao carDao = (CarDao) injector.getInstance(CarDao.class);
```

Comment:
we should use service in controller layer

Reply:
yes,it is significant mistake